### ISSUE TITLE
Msg not sending

### ISSUE THREAD

**Issue by dreamboy-cyber** [Androz2091/discord-logs#139]
Its a nice package 
But some times when the server is boosted .
Bot is not sending msg to channel.

**Comment by Androz2091** [comment]
Yes, I think that this is because the member is boosting twice, right?

**Comment by dreamboy-cyber** [comment]
Yup

**Comment by dreamboy-cyber** [comment]
any solution?

**Comment by BenjammingKirby** [comment]
no it's just not how it works
discord doesn't provide any information or any official event for boosts
internally this checks for <GuildMember>.premiumSince
and while you are boosting and then you boost again, it won't update premiumSince
this can be used to check for members boosting for the first time but that's all that is, it's not foolproof, there isn't a foolproof solution to getting who boosts
i guess `guildUpdate` would emit per boost but you can't narrow down the booster

Tldr; i think the event should be renamed as it is a bit misleading, it's not an actual event for boosting

**Comment by DreamBoy65** [comment]
got it.

**Comment by OmgImAlexis** [comment]
This can be closed, nothing can be done to fix this as discord doesn't give enough info.
